tests/tests/checks/snapshots/web/dirs/check_glob@root_listing__html.snap

```diff
@@ -481,7 +481,7 @@ Spread over multiple lines.
         <tr>
           <td><a href="/tests/source/template_shapes.cpp" class="mimetype-fixed-container mimetype-icon-cpp">template_shapes.cpp</a></td>
           <td class="description"><a href="/tests/source/template_shapes.cpp" title=""></td>
-          <td><a href="/tests/source/template_shapes.cpp">445</a></td>
+          <td><a href="/tests/source/template_shapes.cpp">1126</a></td>
```

Comment:
This change should have been in https://github.com/mozsearch/mozsearch/pull/631. I last ran `review-test-repo` before adding the comment to `template_shapes.cpp`, at which time its size was 445 bytes.

Reply:
Thanks for clarifying / assuming I am this detail oriented ;)